terms.html

```diff
@@ -108,11 +108,9 @@
 a <a>holder</a>.
   </dd>
   <dt><dfn data-lt="party|parties|party's">party</dfn></dt>
-An <a>entity</a> that has 'a mind of its own', which means, among other things, that it 
-An <a>entity</a> that has 'a mind of its own', which means, amongst other things, that it autonomously (sovereignly)
+An <a>entity</a> that has 'a mind of its own', which means, among other things, that it
 <ul>
   <li>creates and maintains (<a>authors</a>) data, including <a>identifiers</a>, <a>claims</a>, and <a>verifiable credentials</a></li>
-  <li>creates and maintains (<a>authors</a>) data, including <a>identifiers</a>, <a>claims</a>, and <a>credentials</a></li>
   <li>decides on the semantics of all data that it <a>authors</a> (i.e., what the data means)</li>
   <li>gathers, stores, processes, and disseminates data</li>
   <li>manages its own logic, deciding what is, and what is not, true, valid, trustworthy, what to reuse (e.g., in terms of shared vocabularies and contexts that other <a>parties</a> have produced), etc.
```

Comment:
Github won't let me touch the line below, which should be `</ul>`, not `</ol>`....
```suggestion
  <li>manages its own logic, deciding what is, and what is not, true, valid, trustworthy, what to reuse (e.g., in terms of shared vocabularies and contexts that other <a>parties</a> have produced), etc.</li>
```